apps/passport-server/src/services/telegramService.ts

```diff
@@ -119,6 +119,8 @@ export class TelegramService {
   private anonBot: Bot<BotContext>;
   private forwardBot: Bot<BotContext> | undefined;
   private rollbarService: RollbarService | null;
+  private readonly MAX_RETRY_ATTEMPTS = 5;
+  private readonly RETRY_INTERVAL_SECONDS = 10;
```

Comment:
let's adjust these numbers to
`MAX_RETRY_ATTEMPTS = 20`
`RETRY_INTERVAL_SECONDS = 30`